docs/admin/auth.md

```diff
@@ -157,3 +163,22 @@ If your OpenID Connect provider requires client TLS certificates for authenticat
 CODER_TLS_CLIENT_CERT_FILE=/path/to/cert.pem
 CODER_TLS_CLIENT_KEY_FILE=/path/to/key.pem
 ```
+
+## Group Sync (enterprise)
+
+If your OpenID Connect provider supports group claims, you can configure Coder
+to synchronize groups in your auth provider to groups within Coder.
+
+To enable group sync, ensure that the `group` claim is set:
```

Comment:
Do some OIDC providers send this claim by default or is this always necessary on the Coder side? Wondering if a deployment might suddenly start doing group sync without the admin realizing (and there happen to be some matching group names in the Coder deployment)?

Reply:
As far as I can tell this is required on the coder side. All of the docs I read don't send group claims by default.